examples/nidcpower_nivisa_dmm_measurement/pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "nidcpower_nivisa_dmm_measurement"
```

Comment:
I don't like this example name. It sounds like it's using both NI-DCPower and NI-VISA to do a DMM measurement, but NI-DCPower is not involved in the DMM measurement.

What is it really doing? 

It's using NI-DCPower to source a DC voltage for the DUT's input pin, and it's using VISA and a DMM to measure the DUT's output pin.

Is there a higher level way to describe this? 

"Stimulus response" comes to mind, but I think that might imply time domain signals, whereas this is working with DC signals.

What sort of DUT would you test with this measurement?

Reply:
As discussed in our PO Connect, I've renamed the measurement to `Output Voltage Measurement`.

Also, I will be creating a separate PR for moving this example into the **multi_instrument** sub-directory with the necessary workflow changes.